spec/support/stubs/rails.rb

```diff
@@ -20,9 +20,17 @@ module Assets
       extend self
     end
 
+    module AssetsManifest
+      extend self
+    end
+
     def assets
       Assets
     end
+
+    def assets_manifest
```

Comment:
These stubs are not ideal. I made another PR that removes them (https://github.com/fphilipe/premailer-rails/pull/202) and makes test suite to test against real Rails app.